packages/file-opener/ios/Plugin/FileOpenerPlugin.swift

```diff
@@ -9,6 +9,7 @@ import Capacitor
 public class FileOpenerPlugin: CAPPlugin {
     public let errorPathMissing = "path must be provided."
     public let errorFileNotExist = "File does not exist."
+    public let errorCouldNotOpen = "Could not open File."
```

Comment:
```suggestion
    public let errorCannotOpenFile = "File cannot be opened."
```

Please use the same variable name and the same error message as on Android.